src/api/transactions/transaction.spec.ts

```diff
@@ -24,6 +29,27 @@ describe("Wiremessages", () => {
+	});
+
+	it("trade ABI test (no fees)", () => {
+		const contract = Address.fromString("0x6e2FD8A6FB6caf3612967AadC749763e9657e9D1");
+		const tradeTx = TypedJSON.parse('{"sig":"0x4fda8964bb4929ed959b8efc6246686c0c851dd1b2f5ff13e7822e6a120544b5547ed6585b25e56602b616b9ac1fc640c387b329178df9bb9f8f0839a6b7c5e11b","sender":"0xcc7b33980df2ed446b3ece6598ddd4a0373ac86f","nonce":"3154030767102693157","offer":{"sig":"0x1935fa389c56ae1deb654b443bd060aa7c58ce97c26ab2bf66f6e9fb85993c1479961b5d96d4c75d80766ae921fb2c81540a5a4686ac37a319e401d031f0a83b1b","owner":"0x853e4adb7133a2428f9110d6fe90528f952c9b8a","offer":{"0x074B0E4cEA85fe6CfEaa1FA5B78aAa137b6C25bA":{"uint":"0x6a2725d52f730ffad9aeaad635512ff952da8971fc39c459e4070a8247cafd86"},"0xBB4e8bD29bbC6c51F7C6204fA131bbF5dE8C7D44":{"uint":"0x9fd4c0687e0f8e85a150bf90b77cbca7f0fec4b9cc8cd97aca91f00e874854f0"},"0xB2F30aCb3D83Eb5310796e3374bE07FcBAeaF8cF":{"uint":"0x178b5ea097edc5fc1ce84a83cb70ada998175f31bb1a81ee1545ec1f27bb9f2f"},"0x0264c08e7381e5275Db666CE3963caD731b71082":{"uint":"0x2f377d8541c4f898abb0d46aad3bc32250270c3faed5a2fc9b66eb57a835d079"}},"expiry":"3510488203188983396","request":{"0x1B8C723CFc1085deBDC8149fea278ddA2CDB7861":{"idset":["0x5208cfd459064675857e2762b5d737508c6ef48c56368cd383a80d6cb5b8ee9c","0xa03135f543d627c14826ab8d9958fb034a332424dae01c6ac25fb18b55cc6c02","0xf93f1b36d53b9f1b53ef620758d817602d6ba4046d3090816af8dc45444645ae"]},"0x63A648217cA03A44A0168d6Ca8aaF633F7c17995":{"uint":"0xd7b959541314b6b5d927cd0fb7bb286594a7e24da98ca89973db17598f0d9e60"},"0xA4608979e027ac444Cc34E2D754EE99E7d2Be147":{"uint":"0x79ede6b960bd5b7b983b935df0cd75c67a4f6335d5da5c2622a77487074ea423"}}}}', Trade)!;
+		expect(tradeTx.offer.fees).to.not.exist;
```

Comment:
Being more specific here with `to.be.undefined` is [recommended](https://www.chaijs.com/api/bdd/#method_exist) and more precise.